market/market/coingecko/coingecko_test.go

```diff
@@ -102,3 +104,58 @@ func Test_normalizeTickers(t *testing.T) {
 		})
 	}
 }
+
+func Test_normalizeTickersWithFilter(t *testing.T) {
+	prices := coingecko.CoinPrices{
+		{
+			Id:           "cUSDC",
+			Symbol:       "cUSDC",
+			CurrentPrice: 0.0021,
+			MarketCap:    minimalMarketCap + 1,
+		},
+		{
+			Id:           "cREP",
+			Symbol:       "cREP",
+			CurrentPrice: 0.02,
+			MarketCap:    minimalMarketCap,
+		},
+		{
+			Id:           "cREP",
+			Symbol:       "cREP",
+			CurrentPrice: 0.02,
+			MarketCap:    minimalMarketCap - 1,
+		},
```

Comment:
I may be misunderstanding how this works - but do we support multiple records for the same symbold from the same provider?

Reply:
Pay attention only to market cap field